trimcase.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require_relative 'decorator'
+# Trim case class
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.slice(0..10)
+  end
+end
```

Comment:
- [ ] This method is currently returning `11` letters rather than `10`, i kindly suggest you adjust your .slice method or see the code snippet below for example :+1:
- code snippet
```ruby
def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```